components/voting.tsx

```diff
@@ -449,30 +498,35 @@ export default class Voting extends React.PureComponent<VotingProps, VotingState
                             <Panel.Heading>
                               <Panel.Title toggle={!this.state.expandAll}>
                                 <SpanIf condition={this.state.expandAll} className="title">
-                                  {this.isVotedFor(s) && (
+                                  {this.isVotedFor(s) && this.state.show !== 'votes' && (
```

Comment:
There's an `isVoting` variable that you could use.

Reply:
Done. OK to merge?